signadot/workspaces/templates/agent.yaml

```diff
@@ -1206,8 +1206,8 @@ spec:
         - /manager
         env:
         - name: REPO_OVERLAY_IMAGE
-          value: signadot/workspaces-repo-overlay:v0.7.10
-        image: signadot/workspaces-operator:v0.7.10
+          value: signadot/workspaces-repo-overlay:v{{ .Chart.AppVersion}}
```

Comment:
We can't actually edit this file here because it's generated by an internal script.

Reply:
oops, thanks and sorry for the noise.  Closing and will try to fix this where it is generated.